src/sparsezoo/validation/integrations.py

```diff
@@ -38,7 +38,12 @@ def validate_nlp():
         "recipe.yaml",
     }
     optional_training_files = {}
-    deployment_files = {"model.onnx", "config.json", "tokenizer.json"}
+    deployment_files = {
+        "model.onnx",
+        "config.json",
+        "tokenizer.json",
+        "tokenizer_config.json",
```

Comment:
does this create a hard check (ie assert/exception) for `tokenizer_config.json` in validation?

Reply:
After code review I think the answer is yes, I'm still working on finding a model to test this hypothesis with.

Reply:
Tested and confirmed, I've removed this line